.gitignore

```diff
@@ -98,3 +98,6 @@ data/*/*.csv
 # Emscripten
 *.o
 web/
+
+# gr-pulsed_power
+!src/gr-pulsed_power/lib
```

Comment:
This was only a workaround for VS Code Source Control plugin. I guess this was an issue from VS Code since now it works properly. I can remove it now.